### ISSUE TITLE
Failure to run on Windows if package.json present

### ISSUE THREAD

**Issue by dandv** [xolvio/chimp#57]
If there's a `package.json` in the directory `node_modules\.bin\chimp --watch` is run from, Chimp fails with this message:

```
[chimp] stderr: 'C:\Program' is not recognized as an internal or external command,
operable program or batch file.

npm install error { [Error: Command failed: C:\Windows\system32\cmd.exe /s /c "C:\Program Files\nodejs\npm install"
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
]
  killed: false,
  code: 1,
  signal: null,
  cmd: 'C:\\Windows\\system32\\cmd.exe /s /c "C:\\Program Files\\nodejs\\npm install"' }
Error in chimp init { [Error: Command failed: C:\Windows\system32\cmd.exe /s /c "C:\Program Files\nodejs\npm install"
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
]
  killed: false,
  code: 1,
  signal: null,
  cmd: 'C:\\Windows\\system32\\cmd.exe /s /c "C:\\Program Files\\nodejs\\npm install"' }
```

To reproduce, save the file below as `package.json` in some directory, then run `npm install` and `node_modules\.bin\chimp --watch`:

```json
{
  "name": "chimp-bug",
  "devDependencies": {
    "chimp": "^0.14.0"
  }
}
```

**Comment by sudsy** [comment]
I was able to partially resolve this on my windows box with a change of the following line in chimp.js:

```
  this.exec(npmPath + ' install', function (error, stdout, stderr) {
```

to

```
  this.exec('"' + npmPath + '" install', function (error, stdout, stderr) {
```

However I hit a node_gyp build error (problem with my windows box) in trying to test the solution so I couldn't verify that the solution really worked.

**Comment by samhatoum** [comment]
thanks for the headsup @sudsy 

I've just updated chimp to work with the node 0.4.1 and it fixed a node_gyp issue I was having. That might sort it for you too. Please try 0.15.x when you get a moment

**Comment by toas1** [comment]
Hi!

We're having the same issue with the latest version of Chimp. Any news on fixing this?

**Comment by samhatoum** [comment]
We haven't worked on it yet. Hoping to get it out in the next release

**Comment by Sanjo** [comment]
This is now fixed with 0.17.2.
